homeassistant/components/gardena/__init__.py

```diff
@@ -0,0 +1,78 @@
+"""
+Support for Gardena Smart sytem devices.
+
+For more details about this component, please refer to the documentation at
+https://home-assistant.io/components/gardena/
+"""
```

Comment:
We no longer include the URl to the docs in the source.

```suggestion
"""Support for Gardena Smart sytem devices."""
```